### ISSUE TITLE
fix: Runtime error: Legacy octal literals are not allowed in strict mode

### ISSUE THREAD

**Issue by Quineone** [nuxt-community/composition-api#142]
**🐛 The bug**
```
Syntax Error: Legacy octal literals are not allowed in strict mode (566:23)
const staticPath = 'E:\1.2-rb2\1.7-subby\rb2-Subby-Template\Frontend\.nuxt\static-json';
```
**🛠️ To reproduce**
`yarn dev`
nuxt version `^2.13.0`

**Comment by danielroe** [comment]
I think this is a duplicate of #137

**Comment by danielroe** [comment]
This should now be fixed in v0.10.6. Let me know if it works for you 😃
